pkg/analysis/passes/metadata/metadata.go

```diff
@@ -48,6 +49,12 @@ func run(pass *analysis.Pass) (interface{}, error) {
 		}
 		return nil, err
 	}
+	var data Metadata
```

Comment:
previously we didn't check if we could unmarshall the metadata before. this led to other validators blindly trying to unmarshall the metadata and getting errors